dashboard/app/models/reference_guide.rb

```diff
@@ -24,6 +24,10 @@ class ReferenceGuide < ApplicationRecord
   validates_uniqueness_of :key, scope: :course_version_id
   validate :validate_key_format
 
+  def course_offering_version
+    "#{course_version.course_offering.key}-#{course_version.key}"
```

Comment:
Currently using this, but maybe it makes more sense to just grab the unitgroup name. Is there an easy way to find that?

Reply:
I don't think we can use UnitGroup name here because it could be either a UnitGroup or a Script. I think sticking with this for now makes sense